src/i18n/rpi-imager_it.ts

```diff
@@ -216,64 +216,64 @@ Aggiungi sia &apos;rpi-imager.exe&apos; che &apos;fat32format.exe&apos; all&apos
     <message>
         <location filename="../DstPopup.qml" line="24"/>
         <source>Storage</source>
-        <translation type="unfinished">Scheda SD</translation>
+        <translation>Storage</translation>
```

Comment:
I don't speak/read Italian, but is it correct for this translation to be the same as the English?

Reply:
If the english is "SD Card" I can translate as "Scheda SD".
But if English is "Storage" I can translates as "Storage" or "Archiviazione".
In this context could be better in Italian as "Storage".